src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -41,6 +41,11 @@ class InstPrepdisk
       def run
         return :auto if Mode.update
 
+        storage = manager.storage
+
+        storage.probed.save(Directory.logdir + "/probed-devicegraph.xml")
+        storage.staging.save(Directory.logdir + "/staging-devicegraph.xml")
+
```

Comment:
A better place to put that lines is `StorageManager#commit` to save logs in all commit calls, not only in the client. Moreover, I think that `Directory` is not defined in that context, so it should be `Yast::Directory`. Does it run without failing? Another thing, would it not be better to save the xml version of the graph directly in the log file instead of create a new file?